tests/test_plot.py

```diff
@@ -17,36 +17,38 @@
 from rasterio.enums import ColorInterp
 
 
-@pytest.mark.skipif(plt is None,
-                    reason="requires matplotlib")
-def test_show_raster():
-    """
-    This test only verifies that code up to the point of plotting with
-    matplotlib works correctly.  Tests do not exercise matplotlib.
-    """
+@pytest.importerskip('matplotlib')
```

Comment:
It's `importorskip` :)

Reply:
lol, as an editor, this kind of kills me :-) fixed in latest push

Reply:
I don't see that the fix made it. 597e4ab is still failing because of `importerskip`.